src/resources/views/pages/profile/profile-info.blade.php

```diff
@@ -4,72 +4,100 @@
-                        <img class="object-cover w-20 h-20" src='https://images.unsplash.com/photo-1438761681033-6461ffad8d80?ixlib=rb-1.2.1&q=80&fm=jpg&crop=entropy&cs=tinysrgb&w=400&fit=max&ixid=eyJhcHBfaWQiOjE0NTg5fQ' alt='Woman looking front'>
-                        </div>
-                    </div>
-                    <div class="grid gap-1">
-                        <div class="col-span-2"></div>
-                        <div class="col-span-2"></div>
-                    <div class="p-profile__sidename">
+<body>
+    <style>
+        .top-negative {
+            top: -205px;
+        }
+    </style>
+    <aside class="p-profile__aside">
+        <ul class="d-dashboard__side-nav">
+            <div class="flex flex-col items-center w-full">
+                <div class="w-20 h-20 border-2 border-white rounded-full overflow-hidden relative">
+                    <img class="object-cover w-20 h-20"
+                        src='https://images.unsplash.com/photo-1438761681033-6461ffad8d80?ixlib=rb-1.2.1&q=80&fm=jpg&crop=entropy&cs=tinysrgb&w=400&fit=max&ixid=eyJhcHBfaWQiOjE0NTg5fQ'
+                        alt='Woman looking front'>
+                </div>
+                <div class="flex flex-col justify-center pl-2 items-center">
+                    <div class="text-sm text-xl font-bold">
                         {{ auth()->user()->profile->first_name }}
                         {{ auth()->user()->profile->middle_name ? auth()->user()->profile->middle_name[0] . '.' : '' }}
                         {{ auth()->user()->profile->last_name }}
                     </div>
-                    <div class="row-span-2 col-span-2 ...">
+                    <div class="text-sm font-normal">
                         {{ auth()->user()->email }}
                     </div>
-                    </div>
                 </div>
-                <div class="p-profile__icons">
+            </div>
+            <hr class="border border-gray-600 mt-2 mb-2 shadow:md w-full">
+            <div class="p-profile__icons">
                 <div class="col-span-2">
                     <li class="d-dashboard__side-list">
-                    <svg class="h-15 w-15 text-black-500"  width="1.3em" height="1.7em" viewBox="0 0 24 24"  fill="none"  stroke="currentColor"  stroke-width="2"  stroke-linecap="round"  stroke-linejoin="round">  <path d="M3 9l9-7 9 7v11a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2z" />  
-                <polyline points="9 22 9 12 15 12 15 22" /></svg>
+                        <svg class="h-15 w-15 text-black-500" width="1.3em" height="1.7em" viewBox="0 0 24 24"
+                            fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
+                            stroke-linejoin="round">
+                            <path d="M3 9l9-7 9 7v11a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2z" />
+                            <polyline points="9 22 9 12 15 12 15 22" />
+                        </svg>
```

Comment:
Create a separate folder named svg and a separate blade file for it so that you can reuse svg to other components or views in the future.

Always do that implementation in every svg

Reply:
Done